third_party/move/move-compiler/tests/move_check/typing/lambda2.move

```diff
@@ -0,0 +1,100 @@
+module 0x8675309::M {
+    // use 0x1::XVector;
+
+    // public inline fun foreach<T>(v: &vector<T>, action: |&T|) { // expected to be not implemented
```

Comment:
I'm adding a comment at the top to try to clarify what we're doing:
```
    // *** NOTE: THIS TEST FILE IS DERIVED FROM lambda.move by commenting out code which has errors
    //     successfully flagged by move-compiler-v2, so that we can check for errors on other lines
    //     which may be shadowed by those errors.
    // 
    //     We keep the commented code so that the error line numbers line up.
    //
    //
    //     ... code removed here to allow above message ...
    //
```
Hopefully that is sufficient.